Podverse/AddToPlaylistViewController.swift

```diff
@@ -64,9 +64,14 @@ class AddToPlaylistViewController: UIViewController {
         self.navigationItem.rightBarButtonItem = new
         
         Playlist.retrievePlaylistsFromServer() { (playlists) -> Void in
-            if let playlists = playlists {
-                self.playlistsArray = playlists
+            var filteredPlaylists:[Playlist] = []
+            
+            for playlist in playlists {
+                if playlist.ownerId == UserDefaults.standard.string(forKey: "userId") {
+                    filteredPlaylists.append(playlist)
```

Comment:
This filteredPlaylists array looks like its not being used after you push all the playlists to it. It looks like its declared above but never used. Also, instead of a for in, you can probably use a .filter() on the playlists variable

Reply:
Oh my word, clearly I didn't test this view at all. Good catch, pushing a fix.